models/Post.php

```diff
@@ -2,23 +2,27 @@
 
 use Db;
 use Url;
-use App;
-use Str;
 use Html;
 use Lang;
 use Model;
 use Markdown;
 use BackendAuth;
-use ValidationException;
-use RainLab\Blog\Classes\TagProcessor;
-use Backend\Models\User;
 use Carbon\Carbon;
-use Cms\Classes\Page as CmsPage;
 use Cms\Classes\Theme;
+use Backend\Models\User;
+use ValidationException;
+use Cms\Classes\Controller;
+use Cms\Classes\Page as CmsPage;
+use RainLab\Blog\Classes\TagProcessor;
+use October\Rain\Database\Traits\Validation;
```

Comment:
What are all of these changes for?

Reply:
Validation was imported to fix `use \October\Rain\Database\Traits\Validation;` which has already been requested to revert above. 
All other imports were there, I only rearranged them (they all now are forming triangle). I can revert it if you don't like it.